doc/content/lfr/cardinal_7pincell/Cardinal_7pin_LFR_demo.md

```diff
@@ -0,0 +1,439 @@
+### UserObjects and Postprocessors style=font-size:125% 
+
+Both `[Postprocessors]` and `[VectorPostprocessors]` can be used to compute quantities of interest from the solution to be included in output or used by other MOOSE applications. With `[UserObjects]`, the users can extract the layer averaged quantities of interested, such as the power density, solid temperature, etc. 
+
+!listing lfr/7pin_cardinal_demo/NT.i start=[UserObjects] end=[Outputs] include-end=False
+
+### Outputs style=font-size:125% 
+
+Finally, the `[Outputs]` block sets the output files from the simulation. Two of the most common options include the exodus and csv file. The Exodus file can be viewed with the same software as the mesh, but now will show the solution and solution derived quantities such as the multi-group scalar flux and power distribution. The csv file stores a summary of the solution. Both are turned on for the Griffin model. The +perf_graph+ parameter is helpful to evaluate the computational run time.
+
+!listing lfr/7pin_cardinal_demo/NT.i block=Outputs 
+
+## MOOSE heat transfer Input Model
+
+The solid tempearture is solved by the MOOSE heat transfer Module here. We define a number of constants at the beginning of the file and set up the mesh from a file.
+
+!listing lfr/7pin_cardinal_demo/HC.i start=half_pinpitch end=[Executioner] include-end=False
+
+The main field to be solved for in the heat transfer model is temperature, which is defined in the `[Variables]`. 
+The governing equations and boundary conditions are specified accordingly in `[Kernels]` and `[BCs]`. 
+
+!listing lfr/7pin_cardinal_demo/HC.i start=[Variables] end=[Materials] include-end=False
+
+The MOOSE heat transfer module will receive power from Griffin in the form of an AuxVariable, so we define a receiver variable for the fission power, as +heat_source+. The MOOSE heat transfer module will also receive a fluid wall temperature from Cardinal as another AuxVariable which we name +nek_surf_temp+. The fluid bluk temperature is also passed from Cardinal to HC for convergence check. Finally, the MOOSE heat transfer module will send the heat flux to Cardinal, so we add a variable named +heat_flux+ that we will use to compute the heat flux using the DiffusionFluxAux auxiliary kernel.
+
+!listing lfr/7pin_cardinal_demo/HC.i block=AuxVariables
+
+!listing lfr/7pin_cardinal_demo/HC.i block=AuxKernels
+
+The coupling and data transfer are handled by blocks `[MultiApps]` and `[Transfers]`. The input file structures are very similar to those described in Griffin input model above, with the main difference is that MOOSE heat transfer Module now uses nekRS as the sub-app. Similarly, `[Postprocessors]` and `[VectorPostprocessors]` are employed to compute additional quantities of interest for solution monitoring and visualization. 
+Finally, we specify a Transient executioner. Because there are no time-dependent kernels in this input file, this is equivalent in practice to using a steady executioner.
```

Comment:
```suggestion
Finally, we specify a +Transient+ executioner. Because there are no time-dependent kernels in this input file, this is equivalent in practice to using a +Steady+ executioner.
```
```suggestion
Finally, we specify a Transient executioner. Because there are no time-dependent kernels in this input file, this is equivalent in practice to using a steady executioner.
```